### ISSUE TITLE
Add support for outbound OPTIONS ping from carrier trunk

### ISSUE THREAD

**Issue by davehorton** [jambonz/jambonz-webapp#362]
We should have an option for outbound gateways to send OPTIONS pings to the remote gateway.  This of course is only valid for sip gateways that are outbound.  There should be a checkbox for each outbound gateway in the webapp labeled "send OPTIONS ping".  If checked, the system (sbc-sip-sidecar) should send an OPTIONS ping every 60 seconds (this should be an env var) to the gateway.  If the gateway fails to respond, or responds with a non-success response, the corresponding sip gateway should be set to inactive (we can leverage the is_active column that already exists in the database).  When a gateway transitions from active to inactive due to a failure of an OPTIONS ping we should write an alert

**Comment by Megamuch** [comment]
I would like to add that a single failure might be too strict for disabling the gateway. I would keep a minimum of 5 missed pings so you really know it is down. 

Another thing to think about is recovery. What happens is the gateway goes down for 1 hour and comes back up. Do we require a manual intervention into jambonz to re-activate the gateway?

**Comment by vdharashive** [comment]
@xquanluu and @davehorton can this feature be completed , its useful feature to keep the trunk up

**Comment by davehorton** [comment]
yes, this requires database changes so we deferred it to 0.8.6 which we will start shortly
